src/index.js

```diff
@@ -12,18 +12,19 @@ const ToLocalStorage = (tasks) => {
 let list;
 let x = 0;
 let y = 0;
+let z = 0;
 function renderTasks() {
   list = JSON.parse(localStorage.getItem('to-do')) || [];
   document.getElementById('table').innerHTML += '';
   list.forEach((items) => {
     if (items.completed) {
       document.getElementById('table').innerHTML += `<tr class="tr check"id=${items.index} >
-          <td><input type="checkbox" class='todo'></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
+          <td><input type="checkbox" class='todo' data-index = "${z++}" ></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
           </td> <td><i class="fa-solid fa-ellipsis-vertical" id="${x++}"></i></td></tr>
           <tr  class="tr">`;
     } else {
       document.getElementById('table').innerHTML += `<tr class="tr"id=${items.index} >
-      <td><input type="checkbox" class='todo'></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
+      <td><input type="checkbox" class='todo' data-index = "${z++}" ></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
```

Comment:
- [ ]  when I mark a task as completed, it does change to true, but when I refresh, even so, it's true, the check box is not displayed as checked, kindly fix that 🙏 

![chekced](https://user-images.githubusercontent.com/89282221/215462005-50d1233e-b00f-4383-babb-f1fbc9d2ba51.gif)

Reply:
> * [ ]   when I mark a task as completed, it does change to true, but when I refresh, even so, it's true, the check box is not displayed as checked, kindly fix that 🙏
> 
> ![chekced](https://user-images.githubusercontent.com/89282221/215462005-50d1233e-b00f-4383-babb-f1fbc9d2ba51.gif)

- [ ] This issue still persists as observed by the previous reviewer and I think if you create a function for the index, it can always be reset.